build/android/scripts/build.sh

```diff
@@ -19,10 +19,6 @@ elif [ $# -eq 1 -o $# -eq 2 ]; then
     else
         API=21
     fi
-    for arch in $ALL_ARCHS; do
-        $0 $NDK_DIR $arch $API
-    done
-    exit 0
```

Comment:
Needed to move this part below otherwise dependencies would be downloaded for every arch